tests/build/depfile_cython.srctree

```diff
@@ -0,0 +1,61 @@
+"""
```

Comment:
`depfile_package_cython` and `depfile_package_cythonize` now seem better names for the two tests.

Reply:
Renamed.